src/System.Management.Automation/resources/FileSystemProviderStrings.resx

```diff
@@ -336,4 +336,7 @@
   <data name="DriveMaxSizeError" xml:space="preserve">
     <value>Maximum size for drive has been exceeded: {0}.</value>
   </data>
-</root>
\ No newline at end of file
+  <data name="ItemExists" xml:space="preserve">
+    <value>Item '{0}' already exists.</value>
```

Comment:
I like that one. Fixed.

I also changed the name of the new string, to be a bit more clear about its use.